### ISSUE TITLE
Figure out how fight health labels will be instantiated

### ISSUE THREAD

**Issue by ColinC5** [utk-cs340-fall22/BattleBeasts#17]
Fraction displaying health remaining for parent character. Position and scale set by script. Responds to incoming damage.

**Comment by ChrisCanaday** [comment]
Moved to Sprint 2

**Comment by ColinC5** [comment]
We can do this similarly to how the Fighter is instantiated into the fight scene. I think what we can do is have the health bars be a child of the fight scene using AddChild(). I think that would allow us to control the health bars easier than if they were children of the fighter instances. This also keeps the fight scene in complete control.

There are multiple good resources to refer to when creating the health bar:
Video for reference for the health bar itself:
https://youtu.be/JYVZRCj1wok (make sure to add a dynamic text element to display "[current health]/[max health]")

Where I learned to instantiate scenes within a scene. check out the 2d one too: https://docs.godotengine.org/en/stable/getting_started/first_3d_game/05.spawning_mobs.html#:~:text=Let%27s%20code%20the%20mob%20spawning%20logic

What I did in Fight.cs where I instantiate two fighters is pretty much how the health bars will work. Just make sure the health bars are connected to one of the fighters. Do this similarly to how I identify which fighter is for the player and which is for the opponent.
